common/pref_names.cc

```diff
@@ -77,6 +78,8 @@ const char kLoadCryptoWalletsOnStartup[] = "brave.wallet.load_on_startup";
 const char kOptedIntoCryptoWallets[] = "brave.wallet.opted_in";
 const char kBinanceAccessToken[] = "brave.binance.access_token";
 const char kBinanceRefreshToken[] = "brave.binance.refresh_token";
+const char kGeminiAccessToken[] = "brave.gemini.access_token";
```

Comment:
these should go in brave/components/gemini/browser/pref_names.h / pref_names.cc

Reply:
fixed